### PR TITLE
Update rubocop configs

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

**Comment by aspiers** [comment]
As per [my comment on the corresponding review in `crowbar-core`](https://github.com/crowbar/crowbar-core/pull/506#issuecomment-227902991), I might be OK with this change, but :-1: until a proper explanation and justification is given in the commit message.  Thanks in advance for the extra info ;-)

File: .hound.ruby.yml [hunk 1]
```diff
@@ -1,13 +1,14 @@
 inherit_from: .hound.suse.yml
 
 AllCops:
-  RunRailsCops: true
+  Rails:
+    Enabled: true
```
Comment (nicolasbock) [inline .hound.ruby.yml]: What are those changes doing? Should those be put into a separate PR?

File: .hound.ruby.yml [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
   DisplayStyleGuide: true
   DisplayCopNames: true
 
 Style/RedundantReturn:
   Exclude:
-    - "chef/data_bags/crowbar/migrate/*"
+    - "chef/data_bags/crowbar/migrate/**/*"
```
Comment (nicolasbock) [inline .hound.ruby.yml]: Same here, separate PR?

File: .hound.ruby.yml (hunk 1 above)
Comment (rsalevsky) [inline .hound.ruby.yml]: Just a syntax change, the old syntax is deprecated.

File: .hound.ruby.yml (hunk 2 above)
Comment (rsalevsky) [inline .hound.ruby.yml]: Same PR but separate commit.

**Comment by MaximilianMeister** [comment]
+1

**Comment by nicolasbock** [comment]
+1
